web/index.php

```diff
@@ -104,4 +111,9 @@ function(Application $app) {
 
 $app->after($app['cors']);
 
-$app->run();
+try {
+    $app->run();
+} catch (Throwable $throwable) {
+    $app[UncaughtErrorHandler::class]->handle($throwable);
+    throw $throwable;
```

Comment:
Nit: Maybe we should add a comment here to clarify why we re-throw (so it shows up in the server logs as well, and so it returns a 500 error).